src/components/ArtistsCollectionView.jsx

```diff
@@ -0,0 +1,35 @@
+import "./ArtistsCollectionView.css";
+import UserMenuButton from "./UserMenuButton";
+import LibraryTabNav from "./LibraryTabNav";
+import LeftArrowButton from "./LeftArrowButton";
+import RightArrowButton from "./RightArrowButton";
+import { PropTypes } from "prop-types";
+import ArtistMediumCardCollection from "./ArtistMediumCardCollection";
+
+export default function ArtistsCollectionView(props) {
+  let { followedArtists } = props;
```

Comment:
This is unused code as indicated by vscode since you are using the prop directly.